reverse-proxy/index.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>YARP Documentation</title>
+    <link rel="shortcut icon" href="//www.microsoft.com/favicon.ico?v2" />
+    <meta http-equiv="refresh" content="2;url=https://aka.ms/YarpDocumentation" />
+    <meta charset="UTF-8">
+    <script type="text/javascript">
+      location.href = 'https://aka.ms/YarpDocumentation';
```

Comment:
Ah, aka.ms doesn't support subpaths so we'd need a new link for every article.

Alternatively, we could point this at `https://dotnet.github.io/yarp/` so relative links continue working.

Reply:
Looks like https://microsoft.github.io/reverse-proxy/articles/getting-started.html isn't getting picked up right now, but it is for https://github.com/microsoft/microsoft.github.io/blob/main/cadl/index.html.
E.g. `https://microsoft.github.io/typespec/foo/bar.html` does forward to `https://typespec.io/foo/bar.html`.